flink/v1.13/build.gradle

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+def flinkProjects = [
+  project(':iceberg-flink:iceberg-flink-1.13'),
+  project(':iceberg-flink:iceberg-flink-1.13-runtime')
+]
+
+configure(flinkProjects) {
+  project.ext {
+    flinkVersion = '1.13.2'
+  }
+}
+
+project(':iceberg-flink:iceberg-flink-1.13') {
+  apply plugin: 'scala'
```

Comment:
Was this accidentally copied from Spark? The build.gradle for Flink doesn't currently use Scala.

Reply:
I diffed the builds and everything looks fine except for this. I think it was probably an accident so I removed it to get the CI to run without it.